google-http-client/src/main/java/com/google/api/client/http/HttpHeaders.java

```diff
@@ -811,6 +815,37 @@ public HttpHeaders setAuthenticate(String authenticate) {
     return this;
   }
 
+  /**
+   * Sets the {@code "Warning"} header or {@code null} for none.
+   *
+   * <p>Overriding is only supported for the purpose of calling the super implementation and
```

Comment:
Is this enforced in code? Could it be, perhaps with a helper method that's protected and making this method final? But that wouldn't allow you to change the return type I suppose.

Reply:
Done.